packages/frontend/src/providers/ActiveJobProvider.tsx

```diff
@@ -40,7 +40,7 @@ export const ActiveJobProvider: FC = ({ children }) => {
     const toastJobStatus = useCallback(
         (job: Job | undefined) => {
             if (job && !isJobsDrawerOpen) {
-                const toastTitle = `${jobStatusLabel(job?.jobStatus).label}`;
+                const toastTitle = jobStatusLabel(job?.jobStatus);
```

Comment:
where did the label disappear from here?

Reply:
sure! `jobStatusLabel` util was returning an object {icon:..., label:...} and `icon` wasn't being used anywhere! Now, `jobStatusLabel` returns the label `string`